flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/util/AWSUtil.java

```diff
@@ -60,10 +59,7 @@
 
 /** Some utilities specific to Amazon Web Service. */
 @Internal
-public class AWSUtil {
-    /** Used for formatting Flink-specific user agent string when creating Kinesis client. */
-    private static final String USER_AGENT_FORMAT = "Apache Flink %s (%s) Kinesis Connector";
-
+public class AWSUtil extends AWSGeneralUtil {
```

Comment:
Is this here for backwards compatibility? Util classes extending util classes seems a bit smelly

Reply:
For backwards compat. since I had to split AWSUtil.